resources/assets/js/components/Application/BasicInfo/BasicInfo.tsx

```diff
@@ -72,6 +74,25 @@ export const BasicInfo: React.FunctionComponent<BasicInfoProps> = ({
       : false,
   };
 
+  const jobEducationReq = localizeField(locale, job, "education");
+  const defaultEducationReq = intl.formatMessage(
+    educationMessages[classification],
```

Comment:
Is this something we should check for before passing it to `intl.formatMessage` in case it doesn't exist?